content/en/getting_started/synthetics/api_test.md

```diff
@@ -122,9 +132,9 @@ The API test and Multistep API test detail pages display an overview of the test
 
 To troubleshoot a failed test, scroll down to **Test Results** and click on a failing test result. Review failed assertions and response details such as status code, response time, and associated headers and body to diagnose the issue.
 
-{{< img src="getting_started/synthetics/api-test-failure.png" alt="API test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/api-test-failure-3.png" alt="API test failure"  style="width:100%;">}}
```

Comment:
Ok just realized this one is bugged (body assertion), let's use https://a.cl.ly/xQu7kwm1 instead.